### ISSUE TITLE
SQL error in ItemList block

### ISSUE THREAD

**Issue by philippniethammer** [Guite/MostGenerator#743]
Activating the generated ItemList block leads to an SQL  exception on display.

```
Uncaught exception SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'OFFSET 0' at line 1 in .../plugins/Doctrine/lib/vendor/Doctrine/DBAL/Connection.php line, 613
#0 .../plugins/Doctrine/lib/vendor/Doctrine/DBAL/Connection.php(613): PDOStatement->execute()
#1 .../plugins/Doctrine/lib/vendor/Doctrine/ORM/Query/Exec/SingleSelectExecutor.php(46): Doctrine\DBAL\Connection->executeQuery('SELECT z0_.id A...', Array, Array)
#2 .../plugins/Doctrine/lib/vendor/Doctrine/ORM/Query.php(249): Doctrine\ORM\Query\Exec\SingleSelectExecutor->execute(Object(Doctrine\DBAL\Connection), Array, Array)
#3 .../plugins/Doctrine/lib/vendor/Doctrine/ORM/AbstractQuery.php(607): Doctrine\ORM\Query->_doExecute()
#4 .../plugins/Doctrine/lib/vendor/Doctrine/ORM/AbstractQuery.php(413): Doctrine\ORM\AbstractQuery->execute(Array, 1)
#5 .../modules/ZelaBilder/lib/ZelaBilder/Entity/Repository/Base/Image.php(748): Doctrine\ORM\AbstractQuery->getResult()
#6 .../modules/ZelaBilder/lib/ZelaBilder/Block/Base/ItemList.php(148): ZelaBilder_Entity_Repository_Base_Image->retrieveCollectionResult(Object(Doctrine\ORM\Query), 'title ASC', true)
#7 [internal function]: ZelaBilder_Block_Base_ItemList->display(Array)
#8 .../lib/util/BlockUtil.php(202): call_user_func(Array, Array)
#9 .../lib/util/BlockUtil.php(164): BlockUtil::show('ZelaBilder', 'ItemList', Array)
#10 .../lib/viewplugins/function.blockposition.php(52): BlockUtil::displayPosition('center', false, true)
#11 .../ztemp/Theme_compiled/Zela/master.php(22): smarty_function_blockposition(Array, Object(Zikula_View_Theme))
#12 .../lib/Zikula/View.php(2897): include('/var/customers/...')
#13 .../lib/Zikula/View.php(729): Zikula_View->_fetch('master.tpl', '', '', false)
#14 .../lib/Zikula/View/Theme.php(319): Zikula_View->fetch('master.tpl', '')
#15 .../index.php(159): Zikula_View_Theme->themefooter()
#16 {main}
```

- Zikula 1.3.10
- MOST CI 0.7.0alpha (Build #1913)
- PHP 5.5.9-1ubuntu4.17
- MySQL 5.6.30-0ubuntu0.14.04.1-log
- Module model: [ZelaBilder.zip](https://github.com/Guite/MostGenerator/files/360132/ZelaBilder.zip)

**Comment by philippniethammer** [comment]
Fix: Block content variable $properties is reused. Paginator got a page size of "null".

See patch for MostGenerator/org.zikula.modulestudio.generator/src/org/zikula/modulestudio/generator/cartridges/zclassic/controller/apis/BlockList.xtend: 
[BlockList.xtend.diff.zip](https://github.com/Guite/MostGenerator/files/360243/BlockList.xtend.diff.zip)

**Comment by Guite** [comment]
Thanks, fixed in 0c613fff5c4276cd37d864e383770166cbb5adaa
